admin/howto/create-billing-account.md

```diff
@@ -0,0 +1,73 @@
+# Provision a paid cloud provider account for 2i2c
+
+For 2i2c to build & manage your JupyterHub, it needs *full* access to a
+cloud provider account on [Google Cloud Platform](https://cloudn.google.com),
+[Amazon Web Services][https://aws.amazon.com], or [Azure](https://azure.microsoft.com).
+This page will provide guidance on how to do that.
+
+## Google Cloud Platform
+
+Currently, Google Cloud Platform (GCP) is 2i2c's preferred cloud provider. 2i2c
+access can be set up in a couple different ways.
+
+### Full billing account access
+
+A GCP [billing account](https://cloud.google.com/billing/docs/how-to/manage-billing-account)
+is attached to a source of funds (Credit Card, Institutional account with invoicing,
+credits provided by Google, etc). Giving 2i2c full access to a billing account
```

Comment:
I feel like the "Giving 2i2c full access..." content should be in a standalone section at the top, since it is more like an "explanation about billing" rather than a step in a how-to (also is provider-agnostic, right?)

Reply:
I tried to write it in provider generic way just now, but I couldn't figure out how - 'billing account' and 'project' are GCP specific. AWS and Azure have other mechanisms

Reply:
Good point - let’s just leave it as is for now